tests/foreman/ui/test_contentview.py

```diff
@@ -1830,115 +1834,548 @@ def test_positive_restart_publish_via_dynflow(self):
-        # As such, all variations in here subject to change.
-        # Variations:
-        #  * Read, Modify,  Promote?, Publish?, Subscribe??
-        """attempt to view content views
+        """Attempt to view content views
 
         @id: ebdc37ed-7887-4f64-944c-f2f92c58a206
 
-        @setup: create a user with the Content View read-only role
+        @setup:
 
-        @assert: User with read-only role for content view can perform all
-        Variations above
-
-        @caseautomation: notautomated
+        1. create a user with the Content View read-only role
+        2. create content view
+        3. add a custom repository to content view
 
+        @assert: User with read-only role for content view can view the
+        repository in the content view
 
         @CaseLevel: Integration
         """
+        repo_name = gen_string('alpha')
+        cv_name = gen_string('alpha')
+        user_login = gen_string('alpha')
+        user_password = gen_string('alphanumeric')
+        role_name = gen_string('alpha')
+        # create a role with content views read only permissions
+        role = entities.Role(name=role_name).create()
+        entities.Filter(
+            organization=[self.organization],
+            permission=entities.Permission(
+                resource_type='Katello::ContentView').search(
+                filters={'name': 'view_content_views'}),
+            role=role,
+            search=None
+        ).create()
+        # create read only products permissions and assign it to our role
+        entities.Filter(
+            organization=[self.organization],
+            permission=entities.Permission(
+                resource_type='Katello::Product').search(
+                filters={'name': 'view_products'}),
+            role=role,
+            search=None
+        ).create()
+        # create a user and assign the above created role
+        entities.User(
+            default_organization=self.organization,
+            organization=[self.organization],
+            role=[role],
+            login=user_login,
+            password=user_password
+        ).create()
+        # create a content view with the main admin account
+        with Session(self.browser) as session:
+            # create a repository
+            # note: the user created above has not access to repositories
+            self.setup_to_create_cv(repo_name=repo_name)
+            # create the first content view
+            make_contentview(
+                session, org=self.organization.name, name=cv_name)
+            self.assertIsNotNone(self.content_views.search(cv_name))
+            # add repository to the created content view
+            self.content_views.add_remove_repos(cv_name, [repo_name])
+        # login as the user created above
+        with Session(self.browser, user_login, user_password):
+            # assert that the user is not an admin user
+            # try to administer users
```

Comment:
this is to assert that the user is not the global admin user, I hoped that this is what you where looking for